docs/source/contributing.rst

```diff
@@ -19,18 +230,31 @@ Our docs are hosted by ReadTheDocs and are built from ReStructuredText (``.rst``
 to HTML files. This build process is done by Sphinx. The Python-side API is
 done by a Sphinx extension named ``sphinx-autodoc``. As of the time of this
 writing, there is no C++-side API docs yet, but in theory Sphinx + Doxygen +
-Breathe allow this functionality. 
+Breathe allow this functionality.
 
 When you edit documentation, you only need to edit the ``.rst`` files in
 docs/source/ and Sphinx will do the rest. If you add a new page, put it in the
 ``docs/source/`` folder, and make sure it's a valid ``.rst`` file. Then link to
 it in the toctree in ``docs/source/index.rst``. (Toctree = table of contents
 tree.) The toctree will preserve the order given.
 
-You can check your work by asking whoever maintains these docs (as of 2022,
-Kevin Fu) to add your branch as a version on the ReadTheDocs dashboard. They
-should go to the webpage, add a version that points to the new doc development
-branch, and build that version, then give you the link.
+You can check your work by building locally or asking whoever maintains these docs (as of 2022, Kevin Fu) to add your branch as a version on the ReadTheDocs dashboard.
+
+**Local Building**
+
+Locally building the docs is a troublesome process to check your branch's changes. Unless you are the person who maintains the docs, it is not worth the trouble. However, this section explains how to locally build our docs.
```

Comment:
this is a little lengthy, you could sum it up as
```suggestion
This section explains how to locally build our docs. Locally building the docs is not worth the trouble unless you don't have access to the RTD dashboard and whoever does can't be reached. 
```

Reply:
I would also move the section below on building via RTD dashboard above this, so people read that first